Userland/Applications/PixelPaint/Tools/EraseTool.cpp

```diff
@@ -53,84 +53,85 @@ void EraseTool::draw_point(Gfx::Bitmap& bitmap, Gfx::Color color, Gfx::IntPoint
     }
 }
 
-GUI::Widget* EraseTool::get_properties_widget()
+ErrorOr<GUI::Widget*> EraseTool::get_properties_widget()
 {
     if (!m_properties_widget) {
-        m_properties_widget = GUI::Widget::construct();
-        m_properties_widget->set_layout<GUI::VerticalBoxLayout>();
+        auto properties_widget = TRY(GUI::Widget::try_create());
+        (void)TRY(properties_widget->try_set_layout<GUI::VerticalBoxLayout>());
```

Comment:
(suspicious api… it seems we never use the return value for this method here. which is somewhat odd. 🧐)

Reply:
iiiii- don’t think so honestly. we are not setting any layout properties like spacing or margin so there’s no need to assign it to a variable.

Reply:
I've just self-yakbaited myself into changing the Layout API. :laughing: